### PR TITLE
{vis}[intel/2016a] NiBabel v2.0.2, PIL v1.1.7, pydicom v0.9.9

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2688.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/dc6de85c7c55f0c3d0fd846ed3daa190 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2147.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/235f434c1374a02998f476bb5c80c98e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2102.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/23ff18d239247477697ffcb45a013e66 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2701.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2a15d5a7588c0eb6451f7f39a77ee8af for a full test report.

File: easybuild/easyconfigs/p/PIL/PIL-1.1.7-intel-2016a-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+patches = ['PIL-%(version)s-find-deps.patch']
+
+pyver = '2.7.11'
+versionsuffix = '-Python-%s' % pyver
+dependencies = [
+    ('zlib', '1.2.8'),
+    ('Python', pyver),
+    ('libjpeg-turbo', '1.4.2'),
+    # currently not used because of dependency hell
+    # ('freetype', '2.6.1'),
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/PIL/PIL-1.1.7-intel-2016a-Python-2.7.11.eb]: should be fixed with `2016a` no ?

File: easybuild/easyconfigs/p/PIL/PIL-1.1.7-intel-2016a-Python-2.7.11.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ This library supports many file formats, and provides powerful image processing and graphics capabilities."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+source_urls = ['http://effbot.org/downloads/']
+sources = ['Imaging-%(version)s.tar.gz']
+
+patches = ['PIL-%(version)s-find-deps.patch']
+
+pyver = '2.7.11'
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/PIL/PIL-1.1.7-intel-2016a-Python-2.7.11.eb]: use the template

File: easybuild/easyconfigs/p/PIL/PIL-1.1.7-intel-2016a-Python-2.7.11.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/p/PIL/PIL-1.1.7-intel-2016a-Python-2.7.11.eb]: yep, will bump to 2.6.3, thx

File: easybuild/easyconfigs/p/PIL/PIL-1.1.7-intel-2016a-Python-2.7.11.eb (hunk 2 above)
Comment (boegel) [inline easybuild/easyconfigs/p/PIL/PIL-1.1.7-intel-2016a-Python-2.7.11.eb]: oops, forgot to clean this up, thx

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
node2615.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ab484034c76bf082e50f86aad67a5008 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
node2601.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/cc73cbc53eb9f5f8deada1296cc0467c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2076.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/cd802f4a8a912fc65802b6c8e9fd2864 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2643.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2328374ec71e80f826ed61387ed77fd2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2076.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/1e6617ac0a948b3ed7a43196687b3e36 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**